tasks/prime.py

```diff
@@ -215,6 +216,65 @@ class PrimeTasks(PrimeDataStorageMixin, ParserTaskMixin, CertTaskMixin, TaskSet)
     tags where appropriate to make filtering for custom tests easier.
     """
 
+    def __init__(self, parent):
+        self.csrf_token = None
+        self.session_token = None
+        super().__init__(parent)
+
+    def customer_path(self, url: str) -> str:
+        return f"{self.user.alternative_host}{url}"
+
+    def on_start(self):
+        self.client.get(self.customer_path("/devlocal-auth/login"))
+        self.csrf_token = self.client.cookies.get("masked_gorilla_csrf")
+        self.client.headers.update({"x-csrf-token": self.csrf_token})
+        resp = self.client.post(
+            self.customer_path("/devlocal-auth/create"),
+            data={"userType": "milmove", "gorilla.csrf.Token": self.csrf_token},
+        )
+        self.session_token = self.client.cookies.get("mil_session_token")
+        if resp.status_code != 200:
+            self.interrupt()
+
+        logged_in_user = self.client.get(self.customer_path("/internal/users/logged_in"))
+        json_resp = logged_in_user.json()
+        service_member_id = json_resp["service_member"]["id"]
+        email = json_resp["email"]
+        user_id = json_resp["id"]
+
+        origin_duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=29"))
+        current_station_id = origin_duty_stations.json()[0]["id"]
+
+        overrides = {
+            "id": service_member_id,
+            "user_id": user_id,
+            "edipi": "9999999999",
+            "personal_email": email,
+            "email_is_preferred": True,
+            "current_station_id": current_station_id,
+        }
+
+        payload = self.fake_request("/service_members/{serviceMemberId}", "patch", INTERNAL_API_KEY, overrides, True)
```

Comment:
This has bit me before in the seed data, if Marine Corps gets chosen for the affiliation that will send it to a different office queue regardless of origin duty station.  That won't affect the prime but just be aware some of those moves will get routed differently and folks may wonder why it disappeared.

Reply:
Thanks for that info! I also see that some of the duty stations have an affiliation associated with them. What is this used for?